config/webpacker.yml

```diff
@@ -36,7 +36,7 @@ development:
   compile: true
 
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  check_yarn_integrity: true
+  check_yarn_integrity: false
```

Comment:
Probably don't need this either.

Reply:
Yes, I suspect that it might be advisable to have this disabled before this PR is ready for review.